ndk/src/main/java/com/bugsnag/android/ndk/NativeBridge.java

```diff
@@ -326,6 +334,10 @@ private void handleStartSession(Object arg) {
         warn("START_SESSION object is invalid: " + arg);
     }
 
+    private void handleStopSession() {
```

Comment:
This method, while consistent with many of the others, is redundant. Could instead just call `stoppedSession()` above, similar to `clearBreadcrumbs()` and `addHandledEvent()`, which also don't need additional processing before crossing to the native layer.